projects/plugins/jetpack/class.jetpack.php

```diff
@@ -6017,6 +6017,10 @@ public function deprecated_hooks() {
 				'replacement' => null,
 				'version'     => 'jetpack-11.8.0',
 			),
+			'jetpack_are_blogging_prompts_enabled'         => array(
+				'replacement' => null,
+				'version'     => 'jetpack-$$next-version$$',
```

Comment:
Unfortunately I don't think our replacement script can handle this just yet.

```suggestion
				'version'     => 'jetpack-11.8.0',
```